src/core/server/saved_objects/migrationsv2/integration_tests/migration_7_13_0_transform_failures.test.ts

```diff
@@ -93,10 +93,28 @@ describe('migration v2', () => {
         '7.14.0': (doc) => doc,
       },
     });
+
+    // registering the `space` type with a throwing migration fn to avoid the migration failing for unknown types
+    coreSetup.savedObjects.registerType({
+      name: 'space',
+      hidden: false,
+      mappings: {
+        properties: {},
+      },
+      namespaceType: 'single',
+      migrations: {
+        '7.14.0': (doc) => {
+          doc.attributes.foo.bar = 12;
+          return doc;
+        },
```

Comment:
Same than for the `cleanup` suite. This new preflight check was causing the suite to fail too early. I'm now registering the `space` suite and registering a failing migration function instead.

Reply:
Maybe we should update test fixtures instead? `Spaces` plugin SO are not relevant for this test anyway.